packages/schema/src/workspace/index.ts

```diff
@@ -149,3 +149,27 @@ export const GlobalSearchResponseSchema = z.object({
     })
   )
 })
+
+export const GetInvitationRequestSchema = PageRequestSchema
+
+const WorkspaceInvitationDetailsSchema = z.object({
```

Comment:
We don't need this to be separate. You can directly put this into `GetWorkspaceInvitationsResponse`